event_waiting_list/templates/waiting_list_registration.xml

```diff
@@ -0,0 +1,108 @@
+                                    </div>
+                                </div>
+                                <div class="form-group form-field o_website_form_required_custom">
+                                    <label class="col-md-3 col-sm-4 control-label" for="email_from">Email</label>
+                                    <div class="col-md-7 col-sm-8">
+                                        <input type="text" class="form-control o_website_form_input" name="email_from" required="" t-att-value="request.params.get('email_from', '')" />
+                                    </div>
+                                </div>
+                                <div class="form-group form-field">
+                                    <label class="col-md-3 col-sm-4 control-label" for="phone">Phone Number</label>
+                                    <div class="col-md-7 col-sm-8">
+                                        <input type="text" class="form-control o_website_form_input" name="phone" t-att-value="request.params.get('phone', '')" />
+                                    </div>
+                                </div>
+                                <div class="form-group">
+                                    <div class="col-md-offset-3 col-sm-offset-4 col-sm-8 col-md-7">
+                                        <span class="btn btn-primary btn-lg o_website_form_send">Register</span>
+                                        <span id="o_website_form_result"></span>
+                                    </div>
+                                </div>
+                            </form>
+                        </div>
+                        <div class="col-md-4 mb32">
+                            <t t-call="website.company_description"/>
+                        </div>
+                    </div>
+                </div>
+                <div class="oe_structure"/>
+            </div>
+        </t>
+    </template>
+    <record id="registered_on_waiting_list" model="website.page">
+        <field name="name">Registered (Waiting list)</field>
+        <field name="type">qweb</field>
+        <field name="url">/registered-waiting-list</field>
+        <field name="website_published">True</field>
+        <field name="key">event_waiting_list.registered_on_waiting_list</field>
+        <field name="arch" type="xml">
+            <t name="Registered (Waiting list)" t-name="event_waiting_list.registered_on_waiting_list">
+                <t t-call="website.layout">
+                    <div id="wrap">
+                        <div class="oe_structure"/>
+                        <div class="container">
+                            <h1>Thanks!</h1>
+                            <div class="row">
+                                <div class="col-md-8">
+                                    <div class="alert alert-success">
+                                        You've been successfully registered on the waiting list for this event.
```

Comment:
don't use a page, use a controller and display data from the event.
Contact info should come from event organizer.